test/src/com/xilinx/rapidwright/util/TestTimingModel.java

```diff
@@ -0,0 +1,46 @@
+package com.xilinx.rapidwright.util;
```

Comment:
Can you place this file into
`test/src/com/xilinx/rapidwright/timing`
to mirror the location of the class you're trying to test please?

Reply:
My bad, I meant to do that.